device/mellanox/x86_64-nvidia_sn5600-r0/ACS-SN5600/sai_5600.xml

```diff
@@ -33,461 +33,457 @@
                 <local-port>1</local-port>
                 <width>8</width>
                 <module>34</module>
-
-                <!-- 0 none, 1=2, 2=4, 3=2,4 -->
                 <breakout-modes>3</breakout-modes>
-
-                <!-- (BITMASK) 2 - 1Gb , 16 - 10Gb , 32 - 40Gb , 384 - 50Gb , 1536 - 100Gb , 4096 - 200Gb, 32768 - 400G, 262144 - 800G -->
```

Comment:
It is better to have the bit mapping at the top of the file. I saw that you removed it and added it below, please move it to upper side of the file